packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -146,13 +148,19 @@ class DraggableScrollableSheet extends StatefulWidget {
   /// The default value is true.
   final bool expand;
 
+  /// A list of target points that the widget should snap to. [minChildSize] and
+  /// [maxChildSize] are implicitly snapped to and should not be specified here-
+  /// pass an empty list to snap the widget between min and max.
+  final List<double>? snapTargets;
+
   /// The builder that creates a child to display in this widget, which will
   /// use the provided [ScrollController] to enable dragging and scrolling
   /// of the contents.
   final ScrollableWidgetBuilder builder;
 
   @override
-  State<DraggableScrollableSheet> createState() => _DraggableScrollableSheetState();
+  State<DraggableScrollableSheet> createState() =>
+      _DraggableScrollableSheetState();
```

Comment:
Revert formatting chnages. This repo does not use dartfmt.

Reply:
Yeah sorry, I know that pollutes the PR and makes it harder to read. I intend to manually undo the formatting changes when the PR is done as I keep hitting the reformat hotkey out of habit.

Reply:
To make the lives of your reviewers easier, please undo the unrelated formatting changes before the next round of reviews.

Reply:
(I didn't fully review the rest because the extra diffs make it unnecessarily painful).

Reply:
Yeah sorry-I just wanted really quick directional feedback (eg right approach wrong approach) before investing a lot more time into the PR (polish, tests etc.). It's not ready for full review, I'll take it out of draft once it is ready for review.

Thanks!